src/integral.jl

```diff
@@ -0,0 +1,21 @@
+# basic integral struct with upper bound and lower bound.
+struct Integral<: Function
+    x
+    ub
+    lb
+    Integral(x,ub,lb) = new(Symbolics.value(x), Symbolics.value(ub), Symbolics.value(lb))
+end
+
+(I::Integral)(x) = Term{SymbolicUtils.symtype(x)}(I, [x])
+(I::Integral)(x::Num) = Num(I(Symbolics.value(x)))
+SymbolicUtils.promote_symtype(::Integral, x) = x
+
+function Base.show(io::IO, I::Integral)
+    print(io, "Integral(", I.x, ")")
+    print(io,"upper_bound: ")
```

Comment:
Use consistent indentation of a space after a comma and no space before the comma.